shell/common/platform_util_linux.cc

```diff
@@ -122,6 +123,18 @@ bool XDGUtil(const std::vector<std::string>& argv,
   // bring up a new terminal if necessary.  See "man mailcap".
   options.environment["MM_NOTTTY"] = "1";
 
+  // If the user set a GDK_BACKEND value of their own, use that,
+  // otherwise unset it becuase Chromium is setting GDK_BACKEND
+  // during GTK initialization and we want to respect user preference.
+  // Setting values in EnvironmentMap to an empty-string
+  // will make sure that they get unset from the environment via
+  // AlterEnvironment().
```

Comment:
Nice job explaining clearly & succinctly that passing in an empty string will clobber the value altogether :+1:

My first thought reading the code was "This isn't unsetting it; it's setting it to an empty string" so it's great explain (a) that's not the case and (b) that it's AlterEnvironment that strips out key/vals where val is an empty string